.ci/unit-tests/RFEM_Toolkit_Test/BHoMDataStructure/Comparer/Section.cs

```diff
@@ -0,0 +1,136 @@
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+using BH.Adapter;
+using BH.Adapter.RFEM6;
+using BH.Engine.Base;
+using BH.Engine.Spatial;
+using BH.oM.Adapters.RFEM6;
+using BH.oM.Adapters.RFEM6.IntermediateDatastructure.Geometry;
+using BH.oM.Analytical.Elements;
+using BH.oM.Data.Requests;
+using BH.oM.Geometry;
+using BH.oM.Physical.Materials;
+using BH.oM.Spatial.ShapeProfiles;
+using BH.oM.Structure.Constraints;
+using BH.oM.Structure.Elements;
+using BH.oM.Structure.MaterialFragments;
+using BH.oM.Structure.SectionProperties;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Principal;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RFEM_Toolkit_Test.Comparer_Tests
+{
+    internal class Section
+
+    {
+
+        RFEM6Adapter adapter;
+        ISectionProperty steelSection1;
+        ISectionProperty steelSection2;
+        ISectionProperty concreteSection0;
+        ISectionProperty concreteSection1;
+        ISectionProperty genericSectionGLTimber;
+        ISectionProperty genericSectionSawnTimber;
+        IProfile rectProfileGLTimber;
+        IProfile circleProfileSawnTimber;
+        IProfile concreteProfile1;
+        IProfile concreteProfile2;
+
+        IMaterialFragment glulam;
+        IMaterialFragment timberC;
+        Concrete concrete0;
+        Concrete concrete1;
+
+        RFEMSectionComparer comparer;
+
+        //[SetUp]
+        //public void Setup()
+        //{
+            
+        //}
+
+        [OneTimeSetUp]
+        public void InitializeRFEM6Adapter()
+        {
+
+            adapter = new RFEM6Adapter(true);
+            comparer = new RFEMSectionComparer();
+
+        }
+
+        [Test]
+        public void StandardConcreteSections()
+        {
+            /***************************************************/
+            /**** Test Preparation                          ****/
+            /***************************************************/
+            concrete0 = BH.Engine.Library.Query.Match("Concrete", "C25/30", true, true).DeepClone() as Concrete;
+            concrete1 = BH.Engine.Library.Query.Match("Concrete", "C45/55", true, true).DeepClone() as Concrete;
+
+
+            concreteProfile1 = BH.Engine.Spatial.Create.CircleProfile(0.2);
```

Comment:
Would call this concreteProfile0 to match with the concreteSection0 name (same for line 97)